src/mono/wasm/debugger/BrowserDebugProxy/DebugStore.cs

```diff
@@ -406,6 +406,31 @@ public MethodInfo(AssemblyInfo assembly, MethodDefinitionHandle methodDefHandle,
             localScopes = pdbMetadataReader.GetLocalScopes(methodDefHandle);
         }
 
+        public List<ParameterInfo> GetMethodParams()
+        {
+            var paramsInfo = new List<ParameterInfo>();
```

Comment:
Keep this as a field, and populate it only the first time.
And this should be an array, since its size is going to be fixed, and known.